.env.test

```
@@ -0,0 +1,14 @@
+DATABASE_URL="mysql://root:fnffn@127.0.0.1:3306/testDevU”
+
+DB_TYPE = mysql
+DB_HOST = 127.0.0.1
+DB_PORT = 3306
+DB_USERNAME = root
+DB_PASSWORD = fnffn
+DB_NAME = testDevU
+
+PORT = 3000
+JWT_SECRET = moladodabnidahahah
+
+ALGORITHM=HS256
+JWT_EXPIRES_IN=1m
```

테스트를 위한 환경변수도 .gitignore 파일에 추가하여 github 상에서 공유가 되지 않도록 해주세요!

이미 github에 push된 파일을 삭제하는 방법은 westagram할 때 부터 많이 가이드드렸던 내용입니다.

git rm --cached "file-path"

테스트 DB에 연동하기 위한 정보도 .env.sample 파일에 명시를 해주면 좋을 것 같습니다.

.env 파일에 추가할 경우 예시
```suggestion
TEST_DATABASE_URL="client://username:password@host-number:port-number/database-name"

TEST_DB_TYPE=DATABASE_CLIENT
TEST_DB_HOST=HOST
TEST_DB_PORT=PORT
TEST_DB_USERNAME=USERNAME
TEST_DB_PASSWORD=PASSWORD
TEST_DB_NAME=TEST_DB
```